src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-configurable-table.svelte

```diff
@@ -0,0 +1,468 @@
+                    Configure Workflow List panel. Click the (</span
+                  ><Icon class="inline" name="add" /><span
+                    >) to add column headings.</span
+                  >
+                </p>
+              </EmptyState>
+            </td>
+          </tr>
+        {/if}
+      </tbody>
+    </table>
+  </div>
+</div>
+
+<Drawer
+  open={customizationDrawerOpen}
+  onClick={closeCustomizationDrawer}
+  position="right"
+  dark={false}
+  title="Configure Workflow List"
+>
+  <svelte:fragment slot="subtitle">
+    <span>Add (</span><Icon class="inline" name="add" /><span
+      >), re-arrange (</span
+    ><Icon class="inline" name="chevron-selector-vertical" /><span
+      >), and remove (</span
+    ><Icon class="inline" name="hyphen" /><span
+      >), Workflow<br />Headings to personalize the Workflow List Table.</span
+    >
+  </svelte:fragment>
+  <div class="flex flex-col gap-4">
+    <OrderableList>
+      <svelte:fragment slot="heading">
+        Workflow Headings <span class="font-normal">(in view)</span>
+      </svelte:fragment>
+      {#each $workflowTableColumns as { label, pinned }, index (label)}
+        <OrderableListItem
+          {index}
+          {pinned}
+          totalItems={$workflowTableColumns.length}
+          maxPinnedItems={MAX_PINNED_COLUMNS}
+          on:moveItem={(event) =>
+            moveColumn(event.detail.from, event.detail.to)}
+          on:pinItem={() => pinColumn(label)}
+          on:removeItem={() => removeColumn(label)}
+        >
+          {label}
+        </OrderableListItem>
+      {:else}
+        <OrderableListItem readonly>No headings in view</OrderableListItem>
+      {/each}
+    </OrderableList>
+    <OrderableList>
+      <svelte:fragment slot="heading">
+        Available Headings <span class="font-normal">(not in view)</span>
+      </svelte:fragment>
+      {#each $availableWorkflowColumns as { label }, index}
+        <OrderableListItem static {index} on:addItem={() => addColumn(label)}>
+          {label}
+        </OrderableListItem>
+      {:else}
+        <OrderableListItem readonly
+          >All available headings are in view</OrderableListItem
+        >
+      {/each}
+    </OrderableList>
+    <OrderableList>
+      <svelte:fragment slot="heading">
+        Custom Search Attributes <span class="font-normal">(not in view)</span>
+      </svelte:fragment>
+      {#each $availableSearchAttributes as searchAttribute, index}
+        <OrderableListItem
+          static
+          {index}
+          on:addItem={() => addColumn(searchAttribute)}
+        >
+          {searchAttribute}
+        </OrderableListItem>
+      {:else}
+        <OrderableListItem readonly
+          >No Custom Search Attributes</OrderableListItem
+        >
+      {/each}
+    </OrderableList>
+  </div>
+</Drawer>
+
+<style lang="postcss">
+  .workflow-summary-tables-wrapper {
+    @apply relative flex flex-row w-full rounded-lg border-primary border-2 overflow-scroll;
```

Comment:
Still not 100% sure if this is what we want here. Users still can't scroll ALL of the table on small screens (only the pinned section), but really not sure who would even be using that table on small screens anyways so not too worried about it.

Reply:
Yeah agreed that this is less than ideal. I'm struggling to come up with a solution that works elegantly across all screen sizes 😕